src/Chirp.Web/Areas/Identity/Pages/Account/Login.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@page
+@model LoginModel
```

Comment:
Maybe we should consider using the internal LoginModel, such that we have more control, of when to implement eventual package updates.